site.yml

```
@@ -76,6 +76,11 @@ apps:
     spec: ojwatson/hrp2malaRia/inst/shiny-seasonality/app
     admin: ojwatson
 
+  nimue_national_fits:
```

Generally urls are better with hyphens than dashes (so this would be shiny.dide.ic.ac.uk/nimue-national-fits)

```suggestion
  nimue-national-fits:
```